core/domain/exp_jobs_one_off.py

```diff
@@ -57,6 +60,32 @@ def map(exploration_model):
     def reduce(exp_id, list_of_exps):
         pass
 
+class ExpSummariesContributorsOneOffJob(jobs.BaseMapReduceJobManager):
+    """ One-off job that finds the user id's of the contributors
+    (defined as any human who has made a positive (i.e. not just
+    a reversion) for each exploration.
+    """
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationSnapshotMetadataModel]
+
+    @staticmethod
+    def map(item):
+        _VERSION_DELIMITER = '-'
```

Comment:
Added that into VersionedModel-- but how do I call it properly from the ExplorationSnapshotMetadataModel/BaseSnapshotMetadataModel? If they were properly inherited (as in if BaseSnapshotMetadataModel was defined as BaseSnapshotMetadataModel(VersionedModel)), then all would be copacetic I think, but with the change you suggest and then replacing that call to rfind with a call to item.get_unversioned_instance_id(), I get the following error:
AttributeError: 'ExplorationSnapshotMetadataModel' object has no attribute 'get_unversioned_instance_id'

Thanks for your input!